### PR TITLE
[5308] Use the same icon for Site Tools in Sidebar and Navigation Menu

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/5308

### CODE REVIEW

File: ui/app/src/components/PublishingStatusTile/PublishingStatusTile.tsx [hunk 1]
```diff
@@ -65,7 +64,7 @@ const usePublishingStatusTileStyles = makeStyles((theme) =>
       ...styles.root
     }),
     avatar: (styles) => ({
-      marginBottom: theme.spacing(1),
+      margin: 5,
```
Comment (rart) [inline ui/app/src/components/PublishingStatusTile/PublishingStatusTile.tsx:67]: What are all these style changes for? including the removal of the padding above.

**Review by rart — APPROVED** [review]

File: ui/app/src/components/PublishingStatusTile/PublishingStatusTile.tsx (hunk 1 above)
Comment (jvega190) [reply ui/app/src/components/PublishingStatusTile/PublishingStatusTile.tsx:67]: The padding/margin was added for the tile (in background variant) to align with the other icons in the launcher. When changing the tile to icon variant it wasn't properly aligned

Comment (rart) [reply ui/app/src/components/PublishingStatusTile/PublishingStatusTile.tsx:67]: Doesn't look like you readjusted the background variant styles to align in that case, @jvega190?

Comment (jvega190) [reply ui/app/src/components/PublishingStatusTile/PublishingStatusTile.tsx:67]: I didn't, I'll adjust it and send it to stability branch

Comment (jvega190) [reply ui/app/src/components/PublishingStatusTile/PublishingStatusTile.tsx:67]: I re-tested and with the latest styles the icon looks aligned in both background and icon variants.

**Pull request merged**